awscli/data/metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+  "distribution_source": "sources"
```

Comment:
For the value here, let's use `source` instead of plural to indicate its a source build. It sort of reads weird by making it plural.